### ISSUE TITLE
Problems of the builder and the courier

### ISSUE THREAD

**Issue by ZzylinN** [ldtteam/minecolonies#1870]
Hello!

I use version 1.12.2-0.8.6037

The builder and can not build a bakery and also requires bread:
http://pixs.ru/showimage/2017121811_3548196_28703983.png

It can be seen that the white inscription on a white background and almost unreadable and also requires bread:
http://pixs.ru/showimage/2017121811_9887143_28703997.png

Here you can see that the builder in the inventory of bread in abundance:
http://pixs.ru/showimage/2017121811_5212349_28704008.png

 In the menu of necessary items, you can see that the bread is enough (the framework was also in the builder's building.) I myself tried to finish the bakery and set the frames where they were supposed to be):
http://pixs.ru/showimage/2017121811_8729947_28704024.png

 

When new people appear, all skills are always equal to 1. Is this how it should be?

 

When delivering items courier to the builder Minecraft freezes. You can move around the map, but do not interact with anything. There is no report on the crash.

**Comment by Optimistos** [comment]
About Builder and bread. When i fullfit its DUBLLICATE in his inventory over 4 stacks of bread 0_o/
Farmer still walking around and tells Hoeing and planting but nothing happens

**Comment by Optimistos** [comment]
У  тебя не все скрины показывает. Not all screenshots is avaible

**Comment by ZzylinN** [comment]
Только что проверил. Все скрины грузятся. Just checked. All screenshots are loaded.

**Comment by Optimistos** [comment]
++

**Comment by MineShadow7** [comment]
С новой системой заказов (Request system) нужно зайти в ее/его инвентарь во вкладку "requests" (eng) и выполнить заказ там.

**Comment by Optimistos** [comment]
Я так делал но работает через раз. Раньше он выдавал бесконечные запросы, сейчас наверное тоже самое но в одном поле, а когда нажимаешь Fill up он их заполняет Фулл стаками.
I did it but works through time. Previously, he gave out infinite requests, now probably the same thing but in one field, and when you press Fill up he fills them with Full stacks.

**Comment by ZzylinN** [comment]
Сейчас вообще на версии 6037 Строитель какой то забагованный или эта ревизия. Если поставить на апгрейт здания (в моем случае это хранилище предметов при запросе дубовых досок строитель вешает намертво Minecraft, т.е. игра как работает, но ни один объект не работает)
И до сих пор строитель не может достроить пекарню (wooden) с 3 до 4 уровня. Не может поставить 2 рамки и воткнуь туда по хлебу. Почему то требует сначала хлеб а потом рамки и не понимает куда воткнуть хлеб.

Now generally on version 6037 the Builder what that zabagovannyj or this revision. If I put on the upgrade of the building (in my case, this store of objects when requesting oak boards the builder hangs tightly Minecraft, ie the game works as it works, but no object works)
And till now the builder can not finish building a bakery (wooden) from 3 to 4 levels. Can not put 2 frames and stick there for bread. Why does it require first bread and then a frame and does not understand where to stick the bread.

**Comment by TheOrionsun** [comment]
From what I see with my game and the issues listed here, these issues have been resolved with the new updates. My bakery was set to repair and it was. It receives grain and it makes bread just fine. However, I have not verified a new build of bakery.

Chat bubble does need to be changed to black text (just where it shows the item they want) if that is possible?

Skills of new people is based on overall happiness and if you have all the different buildings from what I have observed. I had the same concerns when I started my town, all skill were 1, but now they can be much higher.
